eq-author-api/tests/utils/questionnaireBuilder/page/index.js

```diff
@@ -15,4 +17,5 @@ module.exports = {
   duplicatePageMutation,
   movePage,
   movePageMutation,
+  validatePage,
```

Comment:
Does this need to be its own mutation? Can't we just read this information from the `queryPage`?

Reply:
I've modified queryPage test so it can be used for validationError test too. validatePage mutation now removed